core/utils/container_utils.hpp

```diff
@@ -119,13 +124,15 @@ class raw_block_vector_base : private util::noncopyable {
   void pop_buffer() noexcept {
     const auto& bucket = buffers_.back();
     alloc_.deallocate(bucket.data, bucket.size);
+    resource_manager_.Decrease(bucket.size);
     buffers_.pop_back();
   }
 #endif
 
  protected:
   IRS_NO_UNIQUE_ADDRESS std::allocator<byte_type> alloc_;
   std::vector<buffer_t> buffers_;
```

Comment:
Could you add todo about counting this memory too?
I think there is no problem with that, but needs to double check that reallocate still would call just memcpy